CHANGELOG.md

```diff
@@ -6,7 +6,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/bottlerocket-os/bottlerocket-test-system/compare/v0.0.14...develop
+[Unreleased]: https://github.com/bottlerocket-os/bottlerocket-test-system/compare/v0.0.15...develop
+
+## [0.0.15] - 2025-01-30
+
+### Fixed
+
+- fix: add support to create IPv6 clusters ([#957])
```

Comment:
We need to add other commits here as well. It should contain all commits/features/fixes between 0.0.14 to 0.0.15